core/src/main/java/io/confluent/kafka/schemaregistry/storage/SchemaRegistryValue.java

```diff
@@ -15,6 +15,35 @@
 
 package io.confluent.kafka.schemaregistry.storage;
 
-public interface SchemaRegistryValue {
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+import com.fasterxml.jackson.annotation.JsonProperty;
 
+@JsonInclude(Include.NON_EMPTY)
+@JsonIgnoreProperties(ignoreUnknown = true)
+public abstract class SchemaRegistryValue {
+
+  protected Long offset;
```

Comment:
Where are these additional properties getting serialized, some sort of java shorthand i dont understand maybe?

Reply:
They are serialized via the `@JsonProperty` annotations on the accessors below